Tests/UnitTests/Android/app/src/androidTest/java/com/jsruntimehost/unittests/WebSocket.java

```diff
@@ -29,20 +29,20 @@ public void onClose(int code, String reason, boolean remote)
         // For all other close frames, we need to trigger the error callback
         if (code != CloseFrame.NORMAL && code != CloseFrame.ABNORMAL_CLOSE && code != CloseFrame.NEVER_CONNECTED)
         {
-            this.errorCallback();
+            this.errorCallback(reason);
         }
 
-        this.closeCallback();
+        this.closeCallback(code, reason);
     }
 
     @Override
     public void onError(Exception ex)
     {
-        this.errorCallback();
+        this.errorCallback(ex.getMessage());
     }
 
     public native void openCallback();
-    public native void closeCallback();
+    public native void closeCallback(int code, String reason);
```

Comment:
Does this need an update from the new code?

Reply:
I think this was updated right before your comment.